episodes/10-multidimensional-data.md

```diff
@@ -0,0 +1,660 @@
+  clustering objects according to their properties including dimensionality reduction techniques
+  like PCA and UMAP.
+- [napari-acceleration-pixel-and-object-classification](https://www.napari-hub.org/plugins/napari-accelerated-pixel-and-object-classification)
+  for Random Forest-based pixel and object classification.
+- [cellpose-napari](https://www.napari-hub.org/plugins/cellpose-napari)
+  and [stardist-napari](https://www.napari-hub.org/plugins/stardist-napari) for pretrained deep
+  learning-based models to segment cells and nuclei.
+- [napari-n2v](https://www.napari-hub.org/plugins/napari-n2v) for self-supervised deep
+  learning-based denosing.
+- [napari-assistant](https://www.napari-hub.org/plugins/napari-assistant), a meta-plugin for
+  building image processing workflows.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::  callout
+
+## Getting help on image.sc
+
+[image.sc](https://forum.image.sc/) is a community forum for all software-oriented aspects of
+scientific imaging, particularly (but not limited to) image analysis, processing, acquisition,
+storage, and management. Its great place to get help and ask questions which are either general or
+specific to a particular tool. There are active sections
+for [python](https://forum.image.sc/tag/python), [scikit-image](https://forum.image.sc/tag/scikit-image)
+and [Napari](https://forum.image.sc/tag/napari).
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Processing 3D volumetric data
+
+Recall that 3D volumetric data consists of an ordered sequence of images, or slices, each
+corresponding to a specific axial position. As an example lets work with
+`skimage.data.cells3d()`which is a 3D fluorescence microscopy image stack. From
+the [dataset documentation](https://scikit-image.org/docs/stable/api/skimage.data.html#skimage.data.cells3d)
+we can see that the data has two channels (0: membrane, 1: nuclei). The dimensions are ordered
+`(z, channels, y, x)` and each voxel has a size of `(0.29 0.26 0.26)` microns. A voxel is the 3D
+equivalent of a pixel and the size specifies
+the physical dimensions, in `(z, y, x)` order for this example. Note the size of the voxels in z (
+axial) is larger than the voxel spacing in the xy dimensions (lateral). Let's load the data and
+visualise with Napari:
+
+```python
+cells3d = ski.data.cells3d()
+viewer.layers.clear()
+viewer.add_image(data=cells3d, name=["membrane", "nucleus"], channel_axis=1)
+print(cells3d.shape)
+```
+
+```output
+(60, 2, 256, 256)
+```
+
+![](fig/cells3d-slice-napari.png){alt='cell3d slice napari ss'}
+
+Note we now have a dimension slice to control which slice we are visualizing. You can also switch
+to a 3D rendering of the data:
+
+![](fig/cells3d-volume-napari.png){alt='cell3d volume napari ss'}
+
+Many of the scikit-image functions you have used throughout this Lesson work with 3D (or indeed nD)
```

Comment:
Not sure about the capital L for Lesson? Entirely up to you

Reply:
Yes capital L for a Carpentries "Lesson" which is their word for the whole course